shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -920,7 +943,7 @@ \subsection{Block Body Transition}
         \left(
           {\begin{array}{c}
              \bslot{bhb} \\
-             \var{pp} \\
+             \var{dms} \\
```

Comment:
`pp` is used to get the `maxBBSize`

Reply:
good catch! yes, the protocal parameters are inside `ls`, we just need to dig them out :)